sdk/maps/Azure.Maps.Geolocation/tests/Samples/GeolocationClientSamples.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+#region Snippet:GeolocationImportNamespace
+using Azure.Maps.Geolocation;
+using Azure.Maps.Geolocation.Models;
```

Comment:
There shouldn't be any `Azure.Maps.Geolocation.Models` namespace exported to users.